$NetBSD: patch-ad,v 1.1 2010/03/05 14:55:21 gdt Exp $

There is confusion in the pkgsrc-provided Makefile.shlib about what's
a shared library vs a module.  This patch is wrong, but fits with the
existing problems to create a consistent build.  All that really
matters is that the postgis module be dlopenable and everything think
it is in the same place.

--- postgis/Makefile.pgxs.orig	2010-01-12 10:33:49.000000000 +0000
+++ postgis/Makefile.pgxs
@@ -55,7 +55,7 @@ ifdef SCRIPTS_built
 	done
 endif # SCRIPTS_built
 ifdef MODULE_big
-	$(INSTALL_SHLIB) $(shlib) '$(DESTDIR)$(pkglibdir)/$(MODULE_big)$(DLSUFFIX)'
+	$(INSTALL_SHLIB) $(shlib) '$(DESTDIR)$(libdir)/lib$(MODULE_big)$(DLSUFFIX)'
 endif # MODULE_big
 
 installdirs:
